### DISCUSSION TITLE
لطفا اکانت را فیل نماعید چون با پخش فیلم های مستحجن مردم را تحدید میکند

### DISCUSSION THREAD

**Discussion by khanok19** [FILTERING-RUBIKA-RYSON/Filter-Rubika-Ryson#46] [General]
![بارگیری](https://github.com/FILTERING-RUBIKA-RYSON/Filter-Rubika-Ryson/assets/155119323/f726e7eb-1681-40f8-8f1c-0a103095bf28)

**Comment by khanok19** [comment]
لطفا فیل نماعید

**Comment by khanok19** [comment]
سریعا و خیلی فوری این اکانت را فیلتر کنید

**Comment by hosinggo** [comment]
~~_~~_~~~~2~~2
~~~~~~_~~_~~
